### PR TITLE
Timing triggers

### PR DESCRIPTION
**Pull request by ndp-opendap**
Adding BESStopwatch timers to the various parts of the dmr++ serialization processes.

### CODE REVIEW

File: modules/dmrpp_module/Makefile.am [hunk 1]
```diff
@@ -70,7 +70,7 @@ libdmrpp_module_la_LDFLAGS = -avoid-version -module
 libdmrpp_module_la_LIBADD = $(BES_DISPATCH_LIB) $(BES_HTTP_LIB) $(DAP_SERVER_LIBS) $(DAP_CLIENT_LIBS) \
 $(H5_LDFLAGS) $(H5_LIBS) $(OPENSSL_LDFLAGS) $(OPENSSL_LIBS) -ltest-types
 
-bin_PROGRAMS = build_dmrpp check_dmrpp merge_dmrpp reduce_mdf
+bin_PROGRAMS = build_dmrpp check_dmrpp merge_dmrpp reduce_mdf retriever
```
Comment (jgallagher59701) [inline modules/dmrpp_module/Makefile.am:73]: This is where you could make 'retriever' a noinst_program would solve the problem of having the program in the RPM, go through a security audit, etc.

https://gnu.huihoo.org/automake-1.6.1/html_chapter/automake_2.html

The special prefix `noinst' indicates that the objects in question should be built but not installed at all. This is usually used for objects required to build the rest of your package, for instance static libraries (see section 9.2 Building a library), or helper scripts.

File: bes.spec.all_static [hunk 2]
```diff
@@ -201,6 +201,7 @@ exit 0
 %{_bindir}/get_dmrpp
 %{_bindir}/ingest_filesystem
 %{_bindir}/ingest_s3bucket
+%{_bindir}/retriever
```
Comment (jgallagher59701) [inline bes.spec.all_static:204]: If you make the change I suggest further along, I think this can be removed (and the similar line in bes.spec)

**Review by jgallagher59701 — COMMENTED** [review]
Try these changes.

File: bes.spec.all_static (hunk 2 above)
Comment (ndp-opendap) [reply bes.spec.all_static]: Done

**Pull request merged**